ironfish/src/wallet/wallet.ts

```diff
@@ -991,6 +992,16 @@ export class Wallet {
       )
     }
 
+    for (const output of options.outputs ?? []) {
+      if (output.memo.byteLength > MEMO_LENGTH) {
+        throw new Error(
```

Comment:
Make a new error type for this so it can be caught in the RPC in the future. Don't use `Error`